scaletest/templates/scaletest-runner/scripts/prepare.sh

```diff
@@ -51,3 +51,9 @@ log "Cleaning up from previous runs (if applicable)..."
 "${SCRIPTS_DIR}/cleanup.sh" "prepare"
 
 log "Preparation complete!"
+
+log "Scaling up provisioners to ${SCALETEST_PARAM_CREATE_CONCURRENCY}..."
+maybedryrun "$DRY_RUN" kubectl scale deployment/coder-provisioner \
+	--replicas "${SCALETEST_PARAM_CREATE_CONCURRENCY}"
```

Comment:
This is dope! 😍

Reply:
actually I may need to handle concurrency=0 🤔

Reply:
OK, if `SCALETEST_PARAM_CREATE_CONCURRENCY` is 0, I'm defaulting to 10 provisioners.